docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -4,6 +4,160 @@ sidebar_label: With Argo CD
 
 # Advanced Installation with Argo CD
 
+This section outlines the various advanced installation options available for deploying Kargo on a Kubernetes cluster using Argo CD.
```

Comment:
Referring back to [this comment](https://github.com/akuity/kargo/pull/3335/files#r1925603671), this might be an opportunity to provide more clarity.

```suggestion
This section outlines a few generalized approaches to installing and managing Kargo with non-default configuration options using Argo CD.
```